### ISSUE TITLE
Vefiy images Immediately after backup

### ISSUE THREAD

**Issue by Still-Knows-Nothing** [rescuezilla/rescuezilla#307]
Hi,
Would it be possible to have the image you want to backup verified at the same time (just after it has been created)? This would make the whole process much slicker and remove the need to go to the "start" screen and selecting it again? You could ask if the image requires verifying prior to starting the backup and it runs it all in one process.

Many thanks,

**Comment by shasheene** [comment]
Yeah that's easy enough. The confirmation screen is getting a bit cluttered but I'll find a way.

**Comment by Auronius** [comment]
I will vote for adding this option! It will be much more convenient, if backups will be verified just after the creation. Personally I **always** verify backups after they have been created.

**Comment by Still-Knows-Nothing** [comment]
Thanks for the vote! Hopefully it'll be included and it'll make the image verification so much slicker.
